docs/pages/enterprise/sso.mdx

```diff
@@ -199,12 +202,16 @@ spec:
 </TabItem>
 </Tabs>
 
+Note:
+- Manually pasting the XML to `entity_descriptor` field is error prone. We recommend using `tctl sso configure saml -e entity_descriptor.xml ...` command instead.
+- You may use `entity_descriptor_url`, in lieu of `entity_descriptor`, to fetch the entity descriptor from
+  your IDP. Though, we recommend "pinning" the entity descriptor by including the XML rather than fetching from a URL.
```

Comment:
Does the helper command add the XML to pin the entity descriptor? If so, we should indicate this here, since that's the recommended approach.